mle/workflow/report.py

```diff
@@ -26,8 +28,8 @@ def ask_data(data_str: str):
 
 def ask_github_token():
```

Comment:
should we put this util functions here?

very weried to me

Reply:
Yeah, the whole Github auth and token-saving progress is a bit of a mess. We need to refactor it later